bson-bench/README.md

```diff
@@ -0,0 +1,39 @@
+# bson-bench 
+
+This library provides functionality to install and run benchmarks against bson and bson-ext.
+
```

Comment:
@W-A-James TODO: clearly indicate that this is *NOT* a mongodb product and is meant for internal use only